osu.Game/Tests/Visual/OsuManualInputManagerTestScene.cs

```diff
@@ -127,13 +127,13 @@ protected void ClickButtonWhenEnabled<T>()
             where T : Drawable
         {
             if (typeof(T) == typeof(Button))
-                AddUntilStep($"wait for {typeof(T).Name} enabled", () => (this.ChildrenOfType<T>().Single() as Button)?.Enabled.Value == true);
+                AddUntilStep($"wait for {typeof(T).Name} enabled", () => (this.ChildrenOfType<T>().Single() as ClickableContainer)?.Enabled.Value == true);
             else
-                AddUntilStep($"wait for {typeof(T).Name} enabled", () => this.ChildrenOfType<T>().Single().ChildrenOfType<Button>().Single().Enabled.Value);
+                AddUntilStep($"wait for {typeof(T).Name} enabled", () => this.ChildrenOfType<T>().Single().ChildrenOfType<ClickableContainer>().First().Enabled.Value);
 
             AddStep($"click {typeof(T).Name}", () =>
             {
-                InputManager.MoveMouseTo(this.ChildrenOfType<T>().Single());
+                InputManager.MoveMouseTo(this.ChildrenOfType<T>().First());
```

Comment:
Not too sure about this... the reason this was `Single` is to avoid tests potentially clicking a different button as we add more to the interface.

Reply:
Okay, if you're against it I take it you prefer the `ClickButtonWhenEnabled<MultiplayerReadyButton.ReadyButton>()` route?

Reply:
Forced pushed to remove that commit.